src/main/java/frc/robot/subsystems/elevator/factories/RealElevatorConstants.java

```diff
@@ -0,0 +1,97 @@
+package frc.robot.subsystems.elevator.factories;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.SparkLimitSwitch;
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.controller.ElevatorFeedforward;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.robot.constants.IDs;
+import edu.wpi.first.math.filter.Debouncer;
+import frc.robot.hardware.digitalinput.IDigitalInput;
+import frc.robot.hardware.digitalinput.supplied.SuppliedDigitalInput;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.robot.hardware.motor.sparkmax.BrushlessSparkMAXMotor;
+import frc.robot.hardware.motor.sparkmax.SparkMaxDeviceID;
+import frc.robot.hardware.motor.sparkmax.SparkMaxWrapper;
+import frc.robot.hardware.request.cansparkmax.SparkMaxAngleRequest;
+import frc.robot.hardware.signal.supplied.SuppliedAngleSignal;
+import frc.robot.hardware.signal.supplied.SuppliedDoubleSignal;
+import frc.robot.subsystems.elevator.ElevatorConstants;
+import frc.robot.subsystems.elevator.ElevatorMotorStuff;
+import frc.robot.subsystems.elevator.ElevatorStuff;
+import frc.utils.AngleUnit;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.Supplier;
+
+public class RealElevatorConstants {
+
+	private final static double DEBOUNCE_TIME_SECONDS = 0.05;
+
+	private final static Debouncer.DebounceType DEBOUNCE_TYPE = Debouncer.DebounceType.kBoth;
+
+	private final static CANSparkBase.SoftLimitDirection SOFT_LIMIT_DIRECTION = CANSparkBase.SoftLimitDirection.kReverse;
+
+	private final static Rotation2d REVERSE_SOFT_LIMIT_VALUE = Rotation2d.fromRotations(0);
+
+	private final static double ROTATIONS_TO_METERS_CONVERTION_RATIO = 1;
+
+	private final static SparkLimitSwitch.Type REVERSE_LIMIT_SWITCH_TYPE = SparkLimitSwitch.Type.kNormallyOpen;
+
+	public static ElevatorFeedforward FEEDFORWARD_CALCULATOR = new ElevatorFeedforward(0, 0, 0, 0);
```

Comment:
why public

Reply:
the `calibratedFeedForward` command (now static) changes it since it's the only way to effect the func too